python/traverse_test.py

```diff
@@ -287,15 +291,15 @@ def replace(e) -> ElementReplacement:
 
 
 def test_traverse_and_replace_void_elements() -> None:
-    def replace(e) -> ElementReplacement:
+    def replace(e: slp.Node) -> ElementReplacement:
         return e
 
-    html = traverse_and_replace('<div><br><input name="input"</div>', replace)
+    html = traverse_and_replace('<div><br><input name="input"></div>', replace)
```

Comment:
Heh, neat that the old parser was resilient enough to handle this!

Reply:
Yes, sadly the new one isn't quite as robust (the closing `</div>` tags got turned into attributes).

Reply:
Hmm, this actually seems highly undesirable. `lxml` handled this in a somewhat-sensible way, as would our current parser. If selectolax isn't as forgiving, I'm not sure if it's a good idea to move to it, as there are undoubtedly *tons* of existing questions with invalid HTML.

Reply:
I will double check, there might be a way to coax more resilient behavior out of this.

Reply:
So I couldn't get selectolax to be more lenient about this, though I'm not sure how negatively this might affect some questions. Can discuss on Slack, but `lxml` runs into other issues that might be more problematic. Maybe it's worth trying to use the builtin Python HTML parser? Kinda unfortunate all of the libraries I've seen for this seem to be more geared towards web scraping than what we're trying to do.